bin/tapestry-run

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+// const fs = require('fs-extra')
+const path = require('path')
+
+// const Server = require('../src/server')
+// const validator = require('../src/utilities/validator')
+
+const cwd = process.cwd()
+const env = 'production'
+
+// parent project tapestry config, aliased through webpack/client.config.js
+// const configPath = path.resolve(cwd, 'tapestry.config.js')
+// const config = require(configPath)
+
+const serverPath = path.resolve(cwd, '.tapestry/server.bundle.js')
```

Comment:
Could we do check if the file exists and bail out and say something like "run tapestry build first". 

Otherwise I think this PR is 👌 for a first pass.